lib/ui/ui_utils.dart

```diff
@@ -34,13 +34,19 @@ PButton createExtensionButton(String text, String extensionName) {
 
 CoreElement createExtensionCheckBox(String extensionName) {
   final CoreElement input = checkbox();
+  final CoreElement text = span(text: extensionName);
 
   // Disable checkbox for unavailable service extensions.
-  // TODO: grey out text so that the option is more obviously disabled.
-  input.disabled = !serviceManager.serviceExtensionManager
-      .isServiceExtensionAvailable(extensionName);
-  serviceManager.serviceExtensionManager.hasServiceExtension(
-      extensionName, (available) => input.disabled = !available);
+  if (!serviceManager.serviceExtensionManager
+      .isServiceExtensionAvailable(extensionName)) {
+    input.disabled = true;
+    text.clazz('disabled-text');
+  }
+  serviceManager.serviceExtensionManager.hasServiceExtension(extensionName,
+      (available) {
+    input.disabled = !available;
+    text.toggleClass('disabled-text', !available);
```

Comment:
avoid adding the disabled-text class. Instead write a slightly more complex CSS selector
E.g.
input:disabled+label{color:#ccc;}

Reply:
done